docs/source/index.rst

```diff
@@ -1,3 +1,4 @@
+:orphan:
```

Comment:
this disables the first header *How to Read This Document* to be detected for the table of contents

table of contents shall start with the toctree commands below